app/models/event.rb

```diff
@@ -46,7 +47,7 @@ class Event < ApplicationRecord
                   :kid_ticket_price, :cabin_price, :max_adult_tickets_per_request,
                   :max_kid_tickets_per_request, :max_cabins_per_request, :max_cabin_requests,
                   :photo, :photo_cache, :tickets_require_approval, :require_mailing_address,
-                  :allow_financial_assistance, :allow_donations,
+                  :require_role, :allow_financial_assistance, :allow_donations,
```

Comment:
Since this field is set to true by default, but only in the database, I propose to add it to `#before_validation` callback and set it if not set:

```ruby
   def before_validation
     attributes[:require_role] = true if attributes[:require_role].nil?
   end
```

Then the form should show it checked by the default. You'd want to verify that unchecking it saves it as a false.